<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbCameraL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CAMERA LEFT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCameraR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CAMERA RIGHT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStartStop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCalibration">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CALIBRATION</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClose">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>540</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSettings">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SETTINGS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>771</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>КАЛИБРОВКА</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
